### PR TITLE
Updated logo position for Small screens and CSS formatting.

### PR DESCRIPTION
**Pull request by hardikyjoshi**

### CODE REVIEW

File: css/agency.css [hunk 1]
```diff
@@ -792,26 +803,3 @@ img::-moz-selection {
 body {
   webkit-tap-highlight-color: #fed136;
 }
-
-/* Scroll to top of page */
```
Comment (charmaine) [inline css/agency.css]: Why did you remove the scroll-top-top function?

**Review by charmaine — COMMENTED** [review]
Hey @hardikyjoshi! Thanks for the changes (It's always nice to remove inline css 🤒!) Have a look at the comment I left, but I'll be happy to merge it after! 😄

File: css/agency.css (hunk 1 above)
Comment (hardikyjoshi) [reply css/agency.css]: @charmaine Actually previously I was working on some other branch (since I'm a first-time contributor) and later I got to know I was supposed to fork the repo and make changes in that. Apologies. Will add it again and update the PR.

**Comment by charmaine** [comment]
@hardikyjoshi Perfect! Thanks for reviewing the pr and for contributing to this project! 😊 

Feel free to work on anything else in the repo 👍

**Comment by hardikyjoshi** [comment]
@charmaine Sure! 😊
